<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Encoding</class>
 <widget class="QDialog" name="Encoding">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>79</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <widget class="QPushButton" name="encodeButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>475</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Сжать данные</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="browseButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Обзор…</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
